lib/pg_rls/tenant.rb

```diff
@@ -18,7 +18,7 @@ def switch(resource)
       attr_reader :tenant
 
       def fetch
-        @fetch ||= tenant.find_by_tenant_id(
```

Comment:
I found that `@fetch ||=` was caching the value of fetch across different requests, so I've removed it, what do you think about the performance implications of this?

Reply:
So, originally my idea was to cache, the reason behind it is that when a user called multiple times it `tenant_model.current`  would only look for that tenant once, Although I see that, that, could lead to some issue if you trying to switch between tenant under the same connection, to tackle that issue we could add a reset for this action on the `switch` method. Make sense? I'll look forward to your opinion on this matter, maybe I'm missing something


For example, Let's say you have a car controller and you want to load the current tenant preferences on time_zone. you can use `TenantModel.current.time_zone` and then on your wheel controller you want to know if this tenant allows the user to send emails for some crazy reason so you would call `TenantModel.current.send_email?` and it would not run duplicate queries. 

It's the same as device caching the current_user, we could implement a different method and called it current_tenant and use it as an extended concern, but it would be a lot more code since we most adapt the 'tenant' to be eq to the resource given eg; current_admin or current_company

Reply:
That's a nice solution! I'll make the change and test it!